### PR TITLE
#3104 / pager helper can receive new kwargs: symbol_previous, symbol_next, page_alt_text

### PR DESCRIPTION
**Pull request by gleb-rudenko**
Fixes #
Now devs can override pager snippet and extend pager arguments up to 3 new kwargs. To modify prev and next links, and prepend additional text to pager items links (HTML can be used, using literals). 

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [x] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by wardi** [comment]
@gleb-rudenko very nice. This has been an irritant for a few sites I've worked on.

**Comment by gleb-rudenko** [comment]
@wardi , still trying to pass tests 👎

**Comment by gleb-rudenko** [comment]
@wardi , finally, tests are passed, can you take a look please.

File: ckan/templates/user/snippets/pager.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+                q=q,
+                order_by=order_by,
+                symbol_previous=symbol_previous,
+                symbol_next=symbol_next,
+                pager_item_text=pager_item_text
+                )
+            }}
+        </div>
+    {% endif %}
+{% endblock %}
```
Comment (wardi) [inline ckan/templates/user/snippets/pager.html]: isn't this block the same as the one in the extended template? If so, why redefine it? (same question for the other pager snippets)

**Comment by wardi** [comment]
fixes #3104

**Comment by kmbn** [comment]
We decided to close old pull requests that are not actively worked on so that we can focus our effort and attention on PRs made against the current version of CKAN. A lot has changed in the last several months and many PRs will have gone stale or may now be in conflict with the latest version.

Please note: this isn't a judgement about the quality of your pull request. We just don't have the resources to review older PRs.

If you would like to reopen your PR (and update it if necessary), please do!

**Pull request closed without merging**